AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md

```diff
@@ -76,13 +70,13 @@ The sample demonstrates how to:
   - Specifically, the latest Intel® Xeon® Scalable  processors and  Xeon® processors provide hardware acceleration for such tasks.
 - Test the performance of the FP32 model and INT8 (quantization) model.
 
-## Set Environment Variables
+## Prepare the Environment
 
-When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. Set up your CLI environment by sourcing the `setvars` script every time you open a new terminal window. This practice ensures that your compiler, libraries, and tools are ready for development.
+### On Linux* (Only applicable to AI Tools Offline Installer)
```

Comment:
the same issue here - it' not clear what to do if you didn't use offline installer. It seems like this we have this issue in every individual sample README. We just need to come up to standard structure and use it in each README

Reply:
no plan to add guidance for AI Tools selector since it is designed as straight forward tools and users should be able to use it easily. if we still need some guidance for it, we might be better to improve the design.